src/utils/Decorator.tsx

```diff
@@ -17,7 +16,7 @@ export const Decorator = (storyFn: StoryFn<React.ReactNode>) => (
   </ThemeProvider>
 );
 
-export const GuideDecorator = (storyFn: StoryFn<React.ReactNode>) => (
+export const GuideDecorator = (storyFn: any) => (
```

Comment:
For some reason some storybook dependency must have resolved differently and original code errored. Since it's just dev tool and it works, I marked it as any to avoid 2h documentation-diving on how to properly type story function.